src/main/java/org/broadinstitute/consent/http/db/UserDAO.java

```diff
@@ -86,9 +86,8 @@ Integer insertUser(@Bind("email") String email,
                           @Bind("displayName") String displayName,
                           @Bind("createDate") Date createDate);
 
-    @SqlUpdate("update dacuser set email=:email, displayName=:displayName, additional_email=:additionalEmail where dacUserId=:id")
-    void updateUser(@Bind("email") String email,
-                       @Bind("displayName") String displayName,
+    @SqlUpdate("update dacuser set displayName=:displayName, additional_email=:additionalEmail where dacUserId=:id")
```

Comment:
sql nit: We need to standardize the sql when we are touching it. Capitalize keywords and lower case column names:
```suggestion
    @SqlUpdate("UPDATE dacuser SET displayname=:displayName, additional_email=:additionalEmail WHERE dacuserid=:id")
```